spk/syncthing/Makefile

```
@@ -22,7 +23,7 @@ WIZARDS_DIR = src/wizard/
 SERVICE_USER = auto
 SERVICE_SETUP = src/service-setup.sh
 SERVICE_PORT = 8384
-SERVICE_PORT_TITLE = $(DISPLAY_NAME)
+SERVICE_PORT_TITLE = Syncthing GUI
```

```suggestion
SERVICE_PORT_TITLE = Syncthing Web GUI
```

We call it `Syncthing Web GUI` in the service definition (for firewalls), better be consistent here.